webapp/portlet/src/main/webapp/vue-apps/general-settings/components/branding/SiteBranding.vue

```diff
@@ -251,6 +274,11 @@ export default {
         .catch(e => this.errorMessage = String(e))
         .finally(() => this.$root.loading = false);
     },
+    setBorderRadius(value) {
+      if (value >= 0) {
+        this.borderRadius = `${value}px`;
+      }
```

Comment:
Set all time the effective border radius to use, even `0px`

```suggestion
        this.borderRadius = `${value}px`;
```

Reply:
done